ui_framework/components/index.js

```diff
@@ -16,3 +16,8 @@ export {
   KuiConfirmModal,
   KuiModalOverlay
 } from './modal';
+
+export {
+  KuiPager,
+  KuiPagerButtons
```

Comment:
Can we rename this `KuiPagerButtonGroup`, to create an analogy with the `ButtonGroup` component?